config/routes.rb

```diff
@@ -19,4 +18,9 @@
   #temp-testing views
   get "test" => "app#test"
   get "decision/:utorid/:position_id" => "app#decision"
+
+  #mangled links
```

Comment:
explain syntax of mangled links here . ie stuff following /pb is cypertext that has to be looked up somewhere to be useful